src/schema/slack/functions/channel_canvas_create.ts

```diff
@@ -0,0 +1,51 @@
+/** This file was autogenerated. Follow the steps in src/schema/slack/functions/_scripts/README.md to rebuild **/
+import { DefineFunction } from "../../../functions/mod.ts";
+import SchemaTypes from "../../schema_types.ts";
+import SlackTypes from "../schema_types.ts";
+import { SlackPrimitiveTypes } from "../../slack/types/mod.ts";
+
+export default DefineFunction({
+  callback_id: "slack#/functions/channel_canvas_create",
+  source_file: "",
+  title: "Create channel canvas",
+  input_parameters: {
+    properties: {
+      channel_id: {
+        type: SlackTypes.channel_id,
+        description: "Channel name",
+        title: "Select a channel",
+      },
+      canvas_create_type: {
```

Comment:
Similar here, should this field be marked as required?

Reply:
`canvas_create_type` is not required and will default to `blank`